### PR TITLE
[MKP] Fix metadata DB configmap

### PR DESCRIPTION
**Pull request by numerology**
Two fields are missing right now.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2491)

### CODE REVIEW

File: manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/metadata.yaml [hunk 1]
```diff
@@ -138,3 +138,5 @@ data:
   {{ end }}
   mysql_host: "mysql"
   mysql_port: "3306"
+  username: "root"
+  password: ""
```
Comment (rmgogogo) [inline manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/metadata.yaml]: just aware it doesn't support managed storage but it's OK for now.

File: manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/metadata.yaml [hunk 2]
```diff
@@ -138,3 +138,5 @@ data:
   {{ end }}
   mysql_host: "mysql"
   mysql_port: "3306"
+  username: "root"
```
Comment (IronPan) [inline manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/metadata.yaml]: Do you know why pipeline standalone deployment works?
https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/base/metadata/metadata-configmap.yaml#L8

Comment (IronPan) [reply manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/metadata.yaml]: from @numerology lightweight uses secret to store username/pwd which makes sense. 
https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/base/metadata/metadata-mysql-secret.yaml

I'm ok either we merge this in and use secret in subsequent pr or change in this pr.

**Pull request merged**